interfaces/backend.go

```diff
@@ -23,6 +23,47 @@ import (
 	"github.com/snapcore/snapd/snap"
 )
 
+// ConfinementOptions describe confinement configuration.
+//
+// The confinement system controls the initial layout of the mount namespace as
+// well as the set of actions a process is allowed to perform. Confinement is
+// initially defined by the ConfinementType declared by the snap. It can be
+// either "strict", "devmode" or "classic".
+//
+// The "strict" type uses mount layout that puts the base snap as the root
```

Comment:
Ah, good catch. Initially I thought it would be the base snap but until that exists I think we should use core.